models/model.py

```diff
@@ -100,89 +124,76 @@ def get_lr_factor(self, epoch):
         return lr_factor
 
 
-class TransformerClassifier(nn.Module):
-
-    def __init__(self, model_hparams):
-        super().__init__()
-
-        model = model_hparams['model']
-        if model == 'roberta':
-            self.encoder = TransformerModel()
-        else:
-            raise ValueError("Transformer type '%s' is not supported." % model)
-
-        cf_hidden_dim = model_hparams['cf_hid_dim']
-
-        self.classifier = nn.Sequential(
-            # nn.Dropout(model_hparams['dropout']),     # TODO: maybe add later
-            nn.Linear(self.encoder.hidden_size, cf_hidden_dim),
-            nn.ReLU(),
-            nn.Linear(cf_hidden_dim, model_hparams['num_classes'])
-        )
-
-    # needs to be there to catch additional parameter modeL
-    # noinspection PyUnusedLocal
-    def forward(self, batch, **kwargs):
-        inputs, attention_mask = batch['input_ids'], batch['attention_mask']
-
-        out = self.encoder(inputs, attention_mask)
-        out = self.classifier(out)
-        return out, batch['labels']
-
-
-class TransformerModel(nn.Module):
-
+class RobertaEncoder(nn.Module):
```

Comment:
This class should go into its own file `models/roberta_encoder.py` to avoid multi-purpose and long files. Let's use that `models` directory